charts/opentelemetry-demo/templates/_pod.tpl

```diff
@@ -68,24 +68,11 @@ Note: Consider that dependent variables need to be declared before the reference
 {{- end}}
 {{- end }}
 
-{{- if .default.enabled  }}
+{{- if .useDefault.env  }}
+{{ toYaml .defaultValues.env }}
```

Comment:
`defaultValues` is set in `component.yaml` as a way to pass the default values into the pod template.  Due to our values schema, users cannot set it.